### DISCUSSION TITLE
Unable to Proceed with the Installation Process

### DISCUSSION THREAD

**Discussion by mannkizubani** [elementary/installer#686] [Q&A] [answered]
Hi Team, 

I am trying to install **"Elementary OS v7.0 [2023-01-29] RC"** on my Dell Laptop (Inspiron 15). However, during the installation process, when I choose the **"Custom Install (Advanced"** option, the option of **"Erase and Install"** is grayed out. 

I have already created a custom partition table (refer to the screenshots) with four partitions. All the partitions are properly mounted, but still I am failing to proceed with the installation. 

I do not want to go with the default option **"Erase Disk and Install"** as it doesn't allow me to customize the partitions and at the end, I land up with a default partition table.

Can you guys please help me figure out where am I going wrong with the installation process?

**Screenshots:**
![EFI Partition Mounting](https://user-images.githubusercontent.com/10899599/217517080-ddbcc1c4-8a25-44ca-a5d2-ed80c2a71869.jpg)
![Four Partitions Overview](https://user-images.githubusercontent.com/10899599/217517088-4cfa6c35-f640-4d11-9a93-a4c4f9c300a0.jpg)
![Home Partition Mounting](https://user-images.githubusercontent.com/10899599/217517093-a2c09388-609b-4681-99e1-19d88251e1cd.jpg)
![Partition Table](https://user-images.githubusercontent.com/10899599/217517098-f10f3a6c-26b9-4fdd-abf9-c1d8c2f8f3a8.jpg)
![Root (System) Partition Mounting](https://user-images.githubusercontent.com/10899599/217517100-a455a26a-420b-4c8e-9f8e-0972ac6343ca.jpg)
![Screenshot #1](https://user-images.githubusercontent.com/10899599/217517104-8031f469-0698-49b8-b925-dc188300c943.jpg)
![Swap Partition Mounting](https://user-images.githubusercontent.com/10899599/217517109-9c2aad7d-1424-423c-802b-a05bfd21d07d.jpg)

**Comment by lenemter** [comment, marked as answer]
I believe the /boot/efi partition has to be at least 512 mb in size

**Comment by vjr** [comment]
I think the minimum requirement is actually just over 256 MiB and fat32, so
try 300 MB first then a little over 512MB ?

On Wed, 8 Feb 2023 at 17:03, Leo ***@***.***> wrote:

> I believe the /boot/efi partition has to be at least 512 mb in size
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/elementary/discussions/164#discussioncomment-4904092>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAEVPTRLW7FYDT46ITOJH4TWWOAB7ANCNFSM6AAAAAAUVEJ25M>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
"The World is a book, and those who do not travel read only a page." - St.
Augustine.

**Comment by mannkizubani** [comment]
The EFI partition was set to FAT32 only, but increasing its size to 512 GB solved the issue for me, guys. 

Thanks for your quick inputs. 👍🏻  
Appreciate your help 🙂
